server/src/routes/api/v1/saladsRouter.js

```diff
@@ -43,7 +43,17 @@ saladsRouter.get("/:id", async (req, res) => {
     const saladId = req.params.id
     try {
         const showSalad = await Salad.query().findById(saladId)
-        showSalad.reviews = await showSalad.$relatedQuery("reviews")
+        showSalad.user = await showSalad.$relatedQuery("user")
+        const reviews = await showSalad.$relatedQuery("reviews")
+        const reviewsSorted = reviews.sort((b,a) => {
+            return Date.parse(a.createdAt) - Date.parse(b.createdAt)
+        })
```

Comment:
use Objection chainable orderBy to do this while querying the database to save on performance
https://vincit.github.io/objection.js/api/query-builder/find-methods.html#orderby